src/Microsoft.ML.Data/Scorers/MulticlassClassificationScorer.cs

```diff
@@ -379,6 +382,21 @@ public RowImpl(DataViewRow row, DataViewSchema schema)
                     public override ValueGetter<TValue> GetGetter<TValue>(DataViewSchema.Column column) => Input.GetGetter<TValue>(column);
                 }
             }
+
+            #region IDisposable Support
+            private bool _disposed;
+
+            void IDisposable.Dispose()
+            {
+                // TODO: Is it necessary to call the base class Dispose()?
+                if (_disposed)
+                    return;
+
+                (_bindable as IDisposable)?.Dispose();
```

Comment:
`ISchemaBindableMapper` is internal, so if you want you can make it inherit from `IDisposable` without a breaking change.